spec.bs

```diff
@@ -535,17 +535,15 @@ Redemption Records {#redemption-records}
 
 To reduce communication overhead, the browser might cache blobs returned in
 <a http-header>Sec-Private-State-Token</a> header value in redemption responses. These blobs are
-referred as *Redemption Records*. Browsers might choose to store these records
+referred as [=Redemption Records=]. Browsers might choose to store these records
```

Comment:
we probably want to have consistent casing everywhere - but we can fix that in an editorial PR.